openMediaAppearIn/readme.md

```diff
@@ -1,5 +1,5 @@
-# openMediaVideoConferencing.html
-Basic demo of open media conversation routing.
+# openMediaAppearIn.html
+Basic demo of open media conversation routing by glueing [appear.in](https://appear.in/)
```

Comment:
We should should use the 'integrate with' rather than 'glueing" (which is spelled wrong).